recipes-benchmark/s-suite/s-suite_git.bb

```diff
@@ -0,0 +1,21 @@
+SUMMARY = "Small collection of benchmarks for storage I/O"
+LICENSE = "GPLv2"
+LIC_FILES_CHKSUM = "file://COPYING;md5=b529aaa6a0c50f15d29f89609b5c22f3"
+
+SRCREV = "79698f645bfb28d0d966484ddad3a1efb562246d"
+SRC_URI = "git://github.com/Algodev-github/S.git;protocol=https;branch=master"
+
+S = "${WORKDIR}/git"
+
+DEPENDS = "rsync"
```

Comment:
DEPENDS typically goes before SRCREV. in fact the (undocumented?) coding style is to present the variables ordered by 'task'. so depends, then fetch, then compile.. and in the end variables related to packaging such as RDEPENDS or FILES.

Reply:
Also, 'rsync' in DEPENDS is a bit suspect, should it be RDEPENDS?